File: control-flow/Q109/main.go
package main

import "fmt"

func main() {
	var N int
	fmt.Print("Enter N: ")
	fmt.Scan(&N)
	for i := 1; i <= N; i++ {
		fmt.Printf("%d\t%d\t%d\t%d\n", i, i*10, i*100, i*1000)
	}
}

File: control-flow/Q109/main.rs
use std::io::{self, Read};

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).expect("Failed to read input");

    let n: i32 = input
        .split_whitespace()
        .next()
        .expect("No input provided")
        .parse()
        .expect("Not a valid number");

    println!("Enter N:");

    for i in 1..=n {
        println!("{}\t{}\t{}\t{}", i, i * 10, i * 100, i * 1000);
    }
}